src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -366,6 +378,35 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             };
         }
 
+        internal async Task<JObject> GetHostJsonExtensionsAsync()
+        {
+            var hostOptions = _applicationHostOptions.CurrentValue.ToHostOptions();
```

Comment:
take `_applicationHostOptions` as input param to make easier to add unit test

Reply:
Done.